core/src/test/scala/tech/beshu/ror/utils/TestsUtils.scala

```diff
@@ -397,7 +397,7 @@ object TestsUtils {
   }
 
   def getResourcePath(resource: String): Path = {
-    File(getClass.getResource(resource).getPath).path
+    File(getClass.getResource(resource).toURI).path
```

Comment:
On Linux it made no difference if we used getPath or toURI. On Windows we need URI - getPath is Unix-style path.